pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -404,6 +404,22 @@ type KubeletConfigSpec struct {
 
 // KubeletConfigStatus defines the observed state of a KubeletConfig
 type KubeletConfigStatus struct {
+	// Represents the latest available observations of current state.
+	Conditions []KubeletConfigCondition `json:"conditions"`
+}
+
+// KubeletConfigCondition defines the state of the KubeletConfig
+type KubeletConfigCondition struct {
```

Comment:
keeping status object standard will be easy. good example is here https://github.com/openshift/api/blob/4d49b5754a7067c0bf325b67ac6fc5e7d307aaad/config/v1/types_cluster_operator.go#L82-L98

Reply:
Also functions like these might also come in handy :)

Reply:
I cleaned this up in 750d707... Should be good for another review.